build-system/tasks/presubmit-checks.js

```diff
@@ -612,6 +612,14 @@ const forbiddenTerms = {
       'extensions/amp-access/0.1/iframe-api/access-controller.js',
     ],
   },
+  'this\\.skip\\(\\)': {
+    message: 'Use of `this.skip()` is forbidden. Use `this.skipTest()` ' +
+        'from within a `before()` block instead. See #17245.',
+    checkInTestFolder: true,
```

Comment:
Does this only match tests or also source files?

Reply:
It matches all files, but there are no instances of `this.skip()` in source files. I've updated the error message to be more clear.